msteams-platform/get-started/prerequisites.md

```diff
@@ -0,0 +1,204 @@
+
+If you wish to host your app or access resources within Azure, you will need an Azure subscription.  You can [create a free account](https://azure.microsoft.com/free/) before you begin.
+
+## Sign in to your Microsoft 365 and Azure accounts
+
+You will need access to two accounts:
+
+- Your Microsoft 365 account credentials. This is the account that you use to sign in to Teams. If you're using an Microsoft 365 developer program tenant, this is the admin account you set up when you registered for the program.
+- - Your Azure credentials. This is the account that you use to access the Azure Portal and for provisioning new cloud resources to support your app.
+
+# [Visual Studio Code](#tab/vscode)
+
+1. Open Visual Studio Code
+1. Select the Teams icon in the sidebar:
+
+    :::image type="content" source="~/assets/images/teams-toolkit-v2/sidebar-icon.png" alt-text="The Teams Icon in the Visual Studio Code sidebar.":::
+
+1. Select **Sign in to M365**.
+
+    :::image type="content" source="~/assets/images/teams-toolkit-v2/account-commands.png" alt-text="Location of the Accounts section used to sign-in.":::
+
+1. The sign-in process will start using your normal web browser.  Complete the sign-in process for your M365 account.  You will be prompted when you can close the browser and return to Visual Studio Code.
+1. Return to the Teams Toolkit within Visual Studio Code.
+1. Select **Sign in to Azure**.
+
+    > [!TIP]
+    > If you have the Azure Account extension installed and are using the same account, you can skip this step.  You will automatically use the same account as you are using in other extensions.
+
+1. The sign-in process will start using your normal web browser.  Complete the sign-in process for your Azure account.  You will be prompted when you can close the browser and return to Visual Studio Code.
+
+When complete, the **ACCOUNTS** section of the sidebar will show the two accounts separately, together with the number of usable Azure subscriptions available to you.  Ensure you have at least one usable Azure subscription available.  If not, sign out and use a different account.
+
+# [Visual Studio 2019](#tab/vs)
+
+Visual Studio 2019 will prompt you to log in to each service as it is needed.  You do not need to sign in to your M365 and Azure accounts in advance.
+
+# [Command line](#tab/cli)
+
+1. Sign in to Microsoft 365 with the TeamsFX CLI:
+
+    ``` bash
+    teamsfx account login m365
+    ```
+
+    The sign-in process will start using your normal web browser.  Complete the sign-in process for your M365 account.  You will be prompted when you can close the browser.
+
+2. Sign in to Azure with the TeamsFX CLI:
+
+    ``` bash
+    teamsfx account login azure
+    ```
+
+    The sign-in process will start using your normal web browser.  Complete the sign-in process for your Azure account.  You will be prompted when you can close the browser.
+
+The account logins are shared between Visual Studio Code and the TeamsFX CLI.
```

Comment:
just want to double confirm: is the product name `TeamsFx` or `TeamsFX`?

Reply:
I'm using TeamsFX throughout.  Will confirm and fix after launch in another PR.  It isn't important because TeasmFX is only an interim brand before MetaOS branding is confirmed.